packages/amplication-data-service-generator/src/server/create-dotenv.ts

```diff
@@ -65,12 +63,17 @@ function convertToKeyValueSting(arr: VariableDictionary): string {
   if (!arr.length) return "";
   return arr
     .map((item) =>
-      Object.entries(item).map(([key, value]) => `${key}=${value}`)
+      Object.entries(item).map(([key, value]) => {
+        if (!key.trim() || !value.trim())
+          throw new Error("key or value are missing");
```

Comment:
Do not throw errors we can use empty strings - it makes more sense to get the code with a missing parameter instead of failing the entire process - also - it will be very hard for the user to understand the reason for the failure.